.yarn/versions/3dae3775.yml

```diff
@@ -1,16 +1,16 @@
 undecided:
```

Comment:
Do you know if we need that file? If so is there a more apparent name ?

Reply:
This is auto generated from yarn. 

https://yarnpkg.com/getting-started/qa

> .yarn/versions is used by the [version plugin](https://yarnpkg.com/features/release-workflow) to store the package release definitions. **You will want to keep it within your repository.**